docs/development/README.adoc

```diff
@@ -0,0 +1,72 @@
+:toc: macro
+
+= T developer documentation
+
+toc::[]
+
+== <<local-t-network.adoc,Setting up local T network>>
+
+For development purposes, it is convenient to set up a local Ethereum client with
+a group of connected peers. A note-taken walkthrough covers the entire process
+including setting up a local Ethereum client, deploying T contracts and
+configuring T peers.
+
+== Quick installation
+
+To quickly install and start a client use the following scripts.
+
+=== Install scripts
+
+The `+install.sh+` script will:
+
+* install yarn dependencies
+* unlock accounts
+* build contracts
+* deploy contracts
+* build client
+
+The `+initialize.sh+` script will:
+
+* mint & approve `T` tokens
+* stake `T` tokens
+* increase authorization for `RandomBeacon`
+* register operator
+
+Please refer to the scripts `help` for available command arguments and environment variables. `./install.sh --help` or `./initialize.sh --help`. Accounts unlocking and staking can be run outside of these scripts as standalone hardhat tasks.
+
+To deploy and initialize contracts:
+
+* run `./scripts/install.sh`. You need to run it only once.
+
+* run `./scripts/initialize.sh --stake-owner <address>` for the default setup for each client. Default setup assumes that the staking provider, operator, beneficiary, and the authorizer are all have the same addresses as the stake owner. Only the `stake-owner` has to be unique for the default setup. The default stake amount is `1,000,000 T` tokens. The default authorization amount is the minimum authorization amount +
```

Comment:
*are all have* or just *all have*?

Reply:
Also, can we break the line to keep the 80 margin?

Reply:
[6865fcf](https://github.com/keep-network/keep-core/pull/3050/commits/6865fcf9931fd525748866cee3ef41649a67b7b7)